sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,17 @@
+SELECT
+  "genres"."name" as "Genre",
+  COUNT("films"."title") as "Number-of-Films"
+FROM
+  "genres"
+JOIN
+  "filmGenre" USING ("genreId")
+JOIN
+  "films" USING ("filmId")
+JOIN
+  "castMembers" USING ("filmId")
+JOIN
+  "actors" USING ("actorId")
+WHERE
+  "actors"."firstName" = 'Lisa'
```

Comment:
Make sure your results only include Lisa Munroe, not anyone named Lisa:

---

![image](https://user-images.githubusercontent.com/16143771/218934405-b153904b-6872-4a2a-ab0a-903fa6655e79.png)